src/components/Coins.jsx

```diff
@@ -500,7 +500,7 @@ function Coins() {
               <div className="flex justify-between">
                 {selectedCoinData &&
                   selectedCoinData.map((coin) => (
-                    <div className="flex items-center mx-2.5 mt-2">
+                    <div key={coin.id} className="flex items-center mx-2.5 mt-2">

```

Comment:
good for adding a key. Also, check this out to understand why the key is important and when it MUST be unique and when you can use the index as a key. https://www.youtube.com/watch?v=xlPxnc5uUPQ&t=96s

Reply:
Gotcha, thank you Basil